runtime/commonTest/src/kotlinx/serialization/json/JsonParserTest.kt

```diff
@@ -74,4 +74,18 @@ class JsonParserFuzzerTest {
             printResult = false
         )
     }
+
+    @Test
+    fun testTrailingComma() {
+        testTrailingComma("{\"id\":0,}")
+        testTrailingComma("{\"id\":0  ,}")
+        testTrailingComma("{\"id\":0  , ,}")
+    }
+
+
+    private fun testTrailingComma(content: String) {
+        val e = assertFailsWith<JsonParsingException> {  Json.plain.parseJson(content) }
```

Comment:
Make this a `parameterizedTest` to test both streaming and tree parsing?

Reply:
JsonParserTest tests only `parseJson` method, it is not applicable here